RightslineSampleLambdaDotNetV4/RightslineSampleLambdaDotNetV4/Models/EntityRelationshipModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿using RightslineSampleLambdaDotNetV4.Consts;
+using System;
+
+namespace RightslineSampleLambdaDotNetV4.Models
+{
+    public class EntityRelationshipModel
+    {
+        public int? Id { get; set; }
+        public string ParentURL { get; set; }
```

Comment:
I think this is the v3 model, in v4 we moved away from these URL string properties and we also added the template and status of the parent and child